src/telemetry/telemetry.ts

```diff
@@ -20,6 +20,8 @@ export enum TelemetryEventTypes {
     GENEZIO_GENERATE_SDK_ERROR = "GENEZIO_GENERATE_SDK_ERROR",
     GENEZIO_ADD_CLASS = "GENEZIO_ADD_CLASS",
     GENEZIO_DEPLOY_ERROR = "GENEZIO_DEPLOY_ERROR",
+    GENEZIO_PRE_START_LOCAL_SCRIPT = "GENEZIO_PRE_START_LOCAL_SCRIPT",
+    GENEZIO_PRE_RELOAD_LOCAL_SCRIPT = "GENEZIO_PRE_RELOAD_LOCAL_SCRIPT",
```

Comment:
Both of these EventTypes are used on the error side of running the scripts. Please add a `_ERROR` prefix to the event names.